cli/core/hack/apt/build_package.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [ $(uname) != "Linux" ] || [ -z "$(command -v apt)" ]; then
+   echo "This script must be run on a Linux system that uses the APT package manager"
+   exit 1
+fi
+
+# VERSION should be set when calling this script
+if [ -z "${VERSION}" ]; then
+   echo "\$VERSION must be set before calling this script"
+   exit 1
+fi
+
+# Strip 'v' prefix as an apt package version must start with an integer
+VERSION=${VERSION#v}
+
+BASE_DIR=$(cd $(dirname "${BASH_SOURCE[0]}"); pwd)
+OUTPUT_DIR=${BASE_DIR}/_output
+
+# Install build dependencies
+apt-get update
+apt-get install -y dpkg-dev curl
+
+# Copy the tanzu binary to the package directory
+mkdir -p ${OUTPUT_DIR}/tanzu-cli_${VERSION}_linux_amd64/usr/bin
+cp ${BASE_DIR}/../../../../artifacts/linux/amd64/cli/core/v${VERSION}/tanzu-core-linux_amd64 ${OUTPUT_DIR}/tanzu-cli_${VERSION}_linux_amd64/usr/bin/tanzu
+
+# Create the control file
+mkdir -p ${OUTPUT_DIR}/tanzu-cli_${VERSION}_linux_amd64/DEBIAN
+echo "Package: tanzu-cli
+Version: ${VERSION}
+Maintainer: Tanzu CLI project team
+Architecture: all
+Homepage: https://github.com/vmware-tanzu/tanzu-cli/
+Description: The tanzu CLI" \
+      > ${OUTPUT_DIR}/tanzu-cli_${VERSION}_linux_amd64/DEBIAN/control
+
+# Create the .deb package
+dpkg --build ${OUTPUT_DIR}/tanzu-cli_${VERSION}_linux_amd64
```

Comment:
Currently, we only support `amd64` arch. But in future we might have a requirement to support `arm` as well. 
Should we make this script more generic to support different architectures?

Reply:
Good point.  I'll see what I can do.